app/assets/javascripts/upload_setup.js

```diff
@@ -11,6 +11,12 @@ var uploadSetup = function (previewTemplate) {
       window.alert(msg)
     },
     success: function (f) {
+      $('.button--next').removeAttr('disabled')
+
+      var button = $('#click-to-upload')
+      var buttonHtml = button.html()
+      button.html(buttonHtml.replace('Upload documents', 'Upload more documents'))
```

Comment:
Yeah I think it'd be valuable to add some JS testing tooling. Jasmine, chai, etc.